app/src/main/kotlin/io/intrepid/skotlinton/base/BaseFragment.kt

```diff
@@ -1,29 +1,25 @@
 package io.intrepid.skotlinton.base
 
 import android.content.Context
-import android.content.Intent
 import android.os.Bundle
 import android.support.annotation.CallSuper
 import android.support.v4.app.Fragment
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
-
 import butterknife.ButterKnife
 import butterknife.Unbinder
 import io.intrepid.skotlinton.SkotlintonApplication
 import timber.log.Timber
 
-abstract class BaseFragment<P : BaseContract.Presenter> : Fragment(), BaseContract.View {
+@Suppress("AddVarianceModifier")
```

Comment:
I'm not familiar with this check. What are we suppressing here?

Reply:
I would like to have a variance check on the Generic parameter P, but theres a bug with studio at the moment that prevents that. Adding an `out` modifier messes up logging.